djedi/utils/encoding.py

```diff
@@ -1,11 +1,3 @@
-import django
-
-if django.VERSION < (1, 5):
-    from django.utils.encoding import smart_str, smart_unicode
-else:
-    from django.utils.encoding import (
-        smart_bytes as smart_str,
-        smart_text as smart_unicode,
-    )
+from django.utils.encoding import smart_bytes as smart_str, smart_text as smart_unicode
 
 __all__ = ["smart_str", "smart_unicode"]
```

Comment:
Can't we remove this whole file and instead use `smart_bytes` and `smart_text` where it's applicable?

Reply:
Already done that! This file should have been removed, but was re-added in a merge I believe...